src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -501,6 +501,10 @@ data class IndividualUser(
 
   override fun canReadUser(userId: UserId) = isAcceleratorAdmin()
 
+
+
+
+  
```

Comment:
I think these should go

Reply:
Yeah, I was testing a pre-push hook that failed to prevent this from getting pushed! Sorry you had to see it.